.config/.terraform-docs.yml

```diff
@@ -1,10 +1,13 @@
+header-from: ./.config/header.md
+
 version: ""
 
 formatter: markdown table
 
 name: terraform-fortios-address-group-builder
 
 content: |-
+  {{ .Header }}
```

Comment:
@JoseMotto-sg , note that an additional space is required here to maintain the header format:

content: |-
  {{ .Header }}

  # terraform-fortios-address-group-builder

Reply:
done